Fix include path in installed container file to find litehtml.h where it
will be installed.
--- containers/linux/container_linux.h.orig	2022-04-05 15:49:06.000000000 -0500
+++ containers/linux/container_linux.h	2022-06-02 18:58:49.000000000 -0500
@@ -1,7 +1,7 @@
 #ifndef LH_CONTAINER_LINUX_H
 #define LH_CONTAINER_LINUX_H
 
-#include "../../include/litehtml.h"
+#include <litehtml/litehtml.h>
 #include <cairo.h>
 #include <gtkmm.h>
 #include <pango/pangocairo.h>
